package.json

```diff
@@ -2,7 +2,7 @@
   "name": "keysim",
   "version": "1.3.0",
   "description": "Keyboard simulator for JavaScript.",
-  "main": "dist/keysim.js",
+  "main": "lib/keysim.js",
```

Comment:
I think that this makes it a better NPM citizen.

This is what is expected, I think. This is what I usually see.